src/test/emulators/auth/setAccountInfo.spec.ts

```diff
@@ -419,45 +420,41 @@ describeAuthEmulator("accounts:update", ({ authApi, getClock }) => {
   it("should allow creating MFA info", async () => {
     const user = { email: "bob@example.com", password: "notasecret" };
     const { localId, idToken } = await registerUser(authApi(), user);
-
-    const mfaInfo = {
-      mfaEnrollmentId: randomId(28),
-      displayName: "Cell Phone",
-      phoneInfo: TEST_PHONE_NUMBER,
-    };
+    const mfaEnrollmentId = randomId(28);
 
     await authApi()
       .post("/identitytoolkit.googleapis.com/v1/accounts:update")
       .set("Authorization", "Bearer owner")
       .send({
         localId,
         mfa: {
-          enrollments: [mfaInfo],
+          enrollments: [
+            {
+              ...TEST_MFA_INFO,
+              mfaEnrollmentId,
+            },
+          ],
         },
       })
       .then((res) => expectStatusCode(200, res));
 
     const info = await getAccountInfoByIdToken(authApi(), idToken);
     expect(info.mfaInfo).to.have.length(1);
-    const updated = info.mfaInfo?.pop();
-    expect(updated?.displayName).to.eq(mfaInfo.displayName);
-    expect(updated?.phoneInfo).to.eq(mfaInfo.phoneInfo);
-    expect(updated?.mfaEnrollmentId).to.eq(mfaInfo.mfaEnrollmentId);
+    const updated = info.mfaInfo![0];
+    expect(updated.displayName).to.eq(TEST_MFA_INFO.displayName);
+    expect(updated.phoneInfo).to.eq(TEST_MFA_INFO.phoneInfo);
+    expect(updated.mfaEnrollmentId).to.eq(mfaEnrollmentId);
   });
 
   it("should allow adding a second MFA factor", async () => {
-    const mfaInfo = {
-      displayName: "Cell Phone",
-      phoneInfo: TEST_PHONE_NUMBER,
-    };
-    const user = { email: "bob@example.com", password: "notasecret", mfaInfo: [mfaInfo] };
-    const { localId, idToken } = await registerMfaUser(authApi(), user);
-    let info = await getAccountInfoByIdToken(authApi(), idToken);
-    expect(info.mfaInfo).to.have.length(1);
-    const savedMfaInfo = info.mfaInfo![0];
+    const user = { email: "bob@example.com", password: "notasecret", mfaInfo: [TEST_MFA_INFO] };
+    const { localId, idToken } = await registerUser(authApi(), user);
+    const savedUserInfo = await getAccountInfoByIdToken(authApi(), idToken);
+    expect(savedUserInfo.mfaInfo).to.have.length(1);
+    const savedMfaInfo = savedUserInfo.mfaInfo![0];
     const secondMfaFactor = {
       displayName: "Second MFA Factor",
-      phoneInfo: TEST_PHONE_NUMBER,
+      phoneInfo: "+12813308004",
```

Comment:
I appreciate the reference, but let's not put Mike's number in the source code. If you need a different phone number, please create a constant like `TEST_PHONE_NUMBER_2` and stick to the fictional blocks (e.g. the next number after `TEST_PHONE_NUMBER`). I think we'd need three or four of these.